shared/src/business/useCases/caseNote/getUserCaseNoteForCasesInteractor.test.js

```diff
@@ -22,20 +22,23 @@ describe('getUserCaseNoteForCasesInteractor', () => {
 
   const mockJudge = {
     role: ROLES.judge,
+    section: 'colvinChambers',
     userId: 'd7d90c05-f6cd-442c-a168-202db587f16f',
   };
 
   beforeEach(() => {
     mockCurrentUser = mockJudge;
     mockNote = MOCK_NOTE;
 
-    applicationContext.getCurrentUser.mockImplementation(() => mockCurrentUser);
+    applicationContext.getCurrentUser.mockImplementation(
+      () => new User(mockCurrentUser),
+    );
```

Comment:
I don't think we need to mock this any more since we mock the usecase helper below.

Reply:
I believe we still need the mock for the authorization check

Reply:
Oh, duh.  Nothing to see here folks.  Move along, move along.

Reply:
![Alt Text](https://media.giphy.com/media/10RgsuetO4uDkY/giphy.gif)